zds/member/forms.py

```diff
@@ -82,8 +84,9 @@ def confirm_login_allowed(self, user):
                 code="inactive",
             )
         elif not user.profile.is_banned():
+            ban_rationale = Ban.objects.filter(user=user).order_by("-pubdate")[0].note
```

Comment:
Plus adéquat :
```suggestion
            ban_rationale = Ban.objects.filter(user=user).order_by("-pubdate").first().note
```

L'ORM de Django est mieux utilisé comme ça, parce qu'il peut optimiser la requête.